langkit/injections.py

```diff
@@ -27,7 +27,7 @@ def init(model_path: Optional[str] = None):
     )
 
 
-@register_metric_udf(col_name=prompt)
+@register_metric_udf(col_name=_prompt)
```

Comment:
Value error doesn't seem quite right here, though I'm not seeing a built in exception that fits better... maybe `LangKitException`?

Reply:
Agreed. The Value error is more for debugging, whylogs will swallow these.

Let's do in a follow up.

Reply:
https://github.com/whylabs/langkit/issues/55